.rubocop.yml

```diff
@@ -69,7 +69,9 @@ Style/GuardClause:
   Enabled: false
 Style/IfUnlessModifier:
   Enabled: false
-Style/MethodMissing:
+Style/MethodMissingSuper:
+  Enabled: false
```

Comment:
When running rubocop it would complain about this check:

```rake lint
Running RuboCop...
Error: The `Style/MethodMissing` cop has been split into `Style/MethodMissingSuper` and `Style/MissingRespondToMissing`.
(obsolete configuration found in .rubocop.yml, please update it)
RuboCop failed!```

I was getting the same issue on master

Reply:
Possibly from a new version of Rubocop? Whenever I upgrade I always find new default lint errors.